src/components/AccountForm/index.tsx

```diff
@@ -105,13 +117,47 @@ const AccountForm = () => {
     toggleClick('saveAlertModal');
   };
 
-  const onSubmit = (data: object) => {
-    console.log(data);
-    toggleClick('saveAlertModal');
+  const { mutate: profileImageMutation } = useMutation({
+    mutationFn: (value: File) => Users.createImageUrl(value),
+    onSuccess(data) {
+      const { profileImageUrl } = data.data;
+      setProfileFormValue('profileImageUrl', profileImageUrl);
+    },
+  });
+
+  const { setUserData } = useUserStore();
+
+  const { mutate: profileMutation } = useMutation({
+    mutationFn: (value: UsersEditParams) => Users.edit(value),
```

Comment:
매개변수를 전달하는 방법이 조금 다른 것 같습니다!
저는 동일한 매개변수를 사용하기 때문에 아래처럼 작성했는데요, 멘토님께 상황별로 어떻게 사용하는지 여쭤봐도 좋을 것 같습니다!
```suggestion
    mutationFn: Users.edit,
```

Reply:
네 수정해보겠습니다~